go/cmd/master/master.go

```diff
@@ -5,41 +5,80 @@ import (
 	"net/http"
 	"net/rpc"
 	"strconv"
+	"strings"
+	"sync"
 	"time"
 
 	"github.com/namsral/flag"
+	log "github.com/sirupsen/logrus"
 
 	"github.com/PaddlePaddle/Paddle/go/master"
 )
 
+type inMemStore struct {
```

Comment:
`inMemStore` have not much code, but it'll be still more clear if we put it to `inmen_store.go` corresponding to `etcd_store.go`, also the interface `Store`.